src/commcare_cloud/fab/operations/supervisor.py

```diff
@@ -80,8 +80,8 @@ def set_celery_supervisorconf():
             )
             continue
 
-        pooling = params.get('pooling', 'prefork')
-        max_tasks_per_child = params.get('max_tasks_per_child', 50)
+        pooling = params['pooling']
+        max_tasks_per_child = params['max_tasks_per_child']
```

Comment:
is `params` now an instance of `CeleryOptions`?

Reply:
It's currently getting generated here https://github.com/dimagi/commcare-cloud/blob/e0e133bfa4263d168659b494975525d6cda545e9/src/commcare_cloud/fab/fabfile.py#L158 and then set on envs with `env.update(vars)` a few lines below.

once it gets to these lines I'm changing, params is essentially a `CeleryOptions` converted to json (via `.to_json()`).